modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/manager/v1_0/DefaultObjectEntryManagerImpl.java

```diff
@@ -861,15 +862,38 @@ private Map<String, String> _addAction(
 				}
 
 				for (ObjectEntry nestedObjectEntry : nestedObjectEntries) {
+					if (_isFromManySide(
```

Comment:
This name is confusing. What if it is a many to many? it returns false and I am in the many side

Reply:
Make sense @ccorreagg!!